docs/docs/rest-api/public/api/v2/types/resources.raml

```diff
@@ -36,3 +36,32 @@ types:
         description: The amount of GPU cores that are needed for the pod
         example: 1
         default: 0
+
+  ExecutorResources:
+    type: object
+    description: Executor Resource Allocations. Same as Resources but reserved for the pod executor.
+    properties:
+      cpus?:
+        displayName: cpu shares
+        description: The number of CPU shares this pod needs per instance for its executor. This number does not have to be integer, but can be a fraction.
+        type: number
+        minimum: 0.001
```

Comment:
Nope, like discussed with @nfnt . Adapted the minimum values to mesos minimum values.